src/main/java/com/eprosima/fastdds/idl/templates/SerializationTestSource.stg

```diff
@@ -48,23 +50,28 @@ int main(void)
 
 >>
 
-struct_type(ctx, parent, struct) ::= <<
+function_to_run(ctx, parent, struct, cdr_version, test_null_optional) ::= <<
 functions_to_run.emplace_back([]() -> bool
 {
     printf("\n========================================Testing $struct.name$ ========================================\n");
```

Comment:
Maybe this message can be parameterized using the new options `cdr_version` and `test_null_optional` so if there is a failure, it can easily be known which test case failed.